modules/must-gather-flags.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="must-gather-flags_{context}"]
+= Flags available to use with the must-gather command
+
+The flags listed in the following table are available to use with the `oc adm must-gather` command.
+
+.{product-title} Flags for `oc adm must-gather`
+[cols="1,1,3",options="header"]
+|====
+|Flag |Example Command |Use
```

Comment:
Sentence case

Also, we typically use "Description" as the header name for the column with details
```suggestion
|Flag |Example command |Description
```